### DISCUSSION TITLE
How to echo custom fields in swiper

### DISCUSSION THREAD

**Discussion by pavel-neuman** [bootscore/bootscore#244] [Q&A] [answered]
Please, what code should I add to the swiper template in the child theme to be able to display a custom field value in the slider? 

The code below returns errors:

```
<?php if(get_post_meta( $post->ID, 'my_custom_field', true )) : ?>
  <?php echo get_post_meta( $post->ID, 'my_custom_field', true ); ?>
<?php else : ?>
  ...
<?php endif; ?>
```

**Comment by crftwrk** [comment, marked as answer]
May I ask what do you want to do exactly? Are you using ACF?

**Reply by pavel-neuman** [reply]
I want to display specific custom field (regular WP custom field, not ACF), for example if slider_title field has value, display it instead of the_title()

Thanks!

**Reply by pavel-neuman** [reply]
Now I see the problem is that I use the code that works for posts, but I need to use it for pages slider:( Still don't know how to accomplish it

**Reply by pavel-neuman** [reply]
Found a solution:

`                      <?php if ($slider_title = get_post_custom_values($key = 'slider_title')) : ?>
                        <?php echo $slider_title[0]; ?>
                      <?php else : ?>
                        <?php the_title(); ?>
                      <?php endif; ?>`

**Reply by crftwrk** [reply]
Great, solved!
